process-merge-requests

```diff
@@ -346,7 +322,7 @@ def main():
                 else:
                     sh(ns.pre_exec)
             if ns.conf_file:
-                config = ConfigParser.ConfigParser()
+                config = configparser.ConfigParser()
```

Comment:
One minor change needed here:
config = configparser.ConfigParser(strict=False)
This makes it handle duplicate projects being mentioned in different pmr conf files